waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -81,124 +78,30 @@ private ClientCli(String[] args,  boolean useByTest) {
      * 2                            1            1
      * </pre>
      */
-    private static final class Validate extends Cli {
-        private static final String NAME = "validate";
-        private static final String DESCRIPTION = "Submit transactions for validation";
+
+    protected abstract static class ClientCliBaseClass extends Cli {
         private static final String LOCK_NAME = "validate-lock";
         private static final int LOCK_ID = 0;
         private static final int LAMBDA = 1;
-        private static final int DEFAULT_NUMBER_ACTIVE_PARTITIONS = 1;
         private static final Random RANDOM = new Random();
+        protected static final int DEFAULT_NUMBER_ACTIVE_PARTITIONS = 1;
 
         private static final List<PartitionLocalLock> LOCKS = Collections.singletonList(new PartitionLocalLock(LOCK_NAME, LOCK_ID));
```

Comment:
nit: Move it before line 87 so that all `private static final variables` are defined together.